app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,81 @@
+class ProjectsController < ApplicationController
+  responders :flash
+
+  # GET /projects
+  def index
+    @projects = policy_scope(Project).all.sort_by(&:id)
+    @project = Project.new
+    respond_with @projects
+  end
+
+  # GET /projects/1
+  def show
+    authorize project
+    respond_with project
+  end
+
+  # GET /projects/new
+  def new
+    skip_authorization
+
+    @project = Project.new
+    respond_with @project
+  end
+
+  # GET /projects/1/edit
+  def edit
+    authorize project
+    respond_with project
+  end
+
+  # POST /projects
+  def create
+    skip_authorization
+
+    project_id = params[:project][:id]
+
+    unless credential.accessible_project?(project_id)
+      head :forbidden
+      return
+    end
+
+    if Rails.env.development? || Rails.env.test?
+      panoptes_project = { id: project_id, display_name: 'New Project' }
+    else
+      panoptes_project = Effects.panoptes.project(project_id)
```

Comment:
If this is pulling the display name from Panoptes, should we allow it to be updated (in Caesar) via a regular update as below?  The project id is a param, the display name is being pulled, but is subsequently updatable, so neither of them will necessarily correlate to the same Panoptes project.

Reply:
This is a good catch